### ISSUE TITLE
Unable to login to GoDaddy control panel

### ISSUE THREAD

**Issue by ProactiveServices** [kkapsner/CanvasBlocker#598]
Cannot login to GoDaddy control panel with add-on enabled.

## Description
When trying to login to the GoDaddy control panel (ugh) I get a pop-up telling me 
"Your browser is a bit unusual. Try disabling ad blockers and other extensions, enabling javascript, or using a different web browser"
Using a fresh Firefox profile and only installing CanvasBlocker, using default settings reproduces the problem. With CanvasBlocker disabled, login is successful.

## Expected Behavior
Able to login to GoDaddy control panel
Die a little inside that my customer is using GoDaddy

## Current Behavior
Login attempts result in web page reporting that it's unable to login due to an adblocker, javascript or browser fault.

## Possible Solution
Convince customer to move to a reputable web host

## Steps to Reproduce (for bugs)
1. Visit https://www.godaddy.com/ 
2. Click "Sign In" from top-right
3. Click "Sign In" link
4. Enter valid login details

## Context
Tried with a new Firefox profile and installed only CanvasBlocker.

## Your Environment
* Firefox 64-bit 97.0.2 on Kubuntu 20.04
* CanvasBlocker 1.8
* No other addons

## Your Settings
Default settings.

**Comment by kkapsner** [comment]
Can you log in if you whitelist GoDaddy?
Which APIs are protected when visiting the control panel?

**Comment by kkapsner** [comment]
It seems it's the screen API (outerHeight specifically). You can whitelist godaddy for that API and you will be able to log in.

But I have to dig deeper what happens there. Apart of a small bug I found in the code I have to dig deeper to see what they are doing exactly.

**Comment by ProactiveServices** [comment]
Yup I see the same here. Oddly when I was first looking into this, whitelisting didn't appear to work. It does now.

**Comment by kkapsner** [comment]
Their fingerprinting code is highly obfuscated - so no luck until now how they detect that the outerWidth and outerHeight are faked.
